soccer/src/soccer/strategy/agent/position/runner.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <cmath>
+
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp_action/rclcpp_action.hpp>
+#include <spdlog/spdlog.h>
+
+#include <rj_msgs/action/robot_move.hpp>
+
+#include "planning/instant.hpp"
+#include "position.hpp"
+#include "rj_common/field_dimensions.hpp"
+#include "rj_common/time.hpp"
+#include "rj_constants/constants.hpp"
+#include "rj_geometry/geometry_conversions.hpp"
+#include "rj_geometry/point.hpp"
+
+namespace strategy {
+
+/*
+ * This robot runs in a square path
+ */
+class Runner : public Position {
```

Comment:
note the GH actions here and perhaps read up on the rule of 5. you can set many of those to the default implementation, but it's best to explicitly do so